Feature: snacks picker support (fallback) if user doesnt have / want telescope

vvilhelmsen opened the pull request:
Also added description to open_picker so that it shows up on which-key etc

<img width="1334" height="674" alt="image" src="https://github.com/user-attachments/assets/b8e1f006-185b-4b03-af37-d233a7cf41db" />

vvilhelmsen committed:
Feature: snacks picker support (fallback) if user doesnt have / want telescope

Also added description to open_picker so that it shows up on which_keys etc

vvilhelmsen commented:
Also; feel free to disregard my PR if this isn't something you're interested in for your project. This is just what I use myself since I don't want to have both telescope and snacks picker installed at the same time :)

45Hnri commented:
Hi @vvilhelmsen thanks for the PR and sorry I did not reply to you sooner. Looks like Github doesn't notify me for PRs.
I'll take a look.

integeresting committed:
refactor: isolating some common concerns

Separated picker independent action behavior.
Added some useful fields to utils.

45Hnri commented:
Ok, I reviewed everything. Made some changes extracting common concerns.

I'll merge them ASAP.

Currently blocked by my own incompetence to push changes to the PR
